sui_core/src/authority/authority_store.rs

```diff
@@ -1046,6 +1055,23 @@ impl<
         Ok(transaction)
     }
 
+    pub fn insert_new_enpoch_info(&self, epoch_info: EpochInfo) -> SuiResult {
+        self.epochs
+            .insert(&epoch_info.committee.epoch(), &epoch_info)?;
+        Ok(())
+    }
+
+    pub fn get_last_epoch_info(&self) -> SuiResult<EpochInfo> {
```

Comment:
Should we allow for an optional epoch ID and if None, it returns latest, if some it returns that one?